client/src/game/reducers/noteIdentity.ts

```diff
@@ -78,6 +170,9 @@ function getPossibilitiesFromKeyword(
   variant: Variant,
   keywordPreTrim: string,
 ): Array<[number, number]> | null {
+  if (1) {
```

Comment:
Remove whole function

Reply:
if you dont want the function there, then comment it out, right?
or better yet just delete it entirely if the code isn't being used